extratrees/src/main/resources/assets/extratrees/lang/zh_CN.lang

```diff
@@ -0,0 +1,686 @@
+extratrees.block.planks.yew.desc=一种坚硬的树，带着十分吸引人的颜色。|伐木工
+extratrees.block.planks.pear.desc=不像苹果木，梨木并不是那么独特。|伐木工
+extratrees.block.planks.hawthorn.desc=一种质地均匀的软木。|伐木工
+extratrees.block.planks.rowan.desc=一种颜色柔和的木材，木板之间带有好看的色差。|伐木工
+extratrees.block.planks.elder.desc=一种质地均匀的木材，却不像其它木板一样吹得玄乎其玄。|伐木工
+extratrees.block.planks.maclura.desc=桑橙树木质那浓郁的橙色使它很令人满意。|伐木工
+extratrees.block.planks.syzgium.desc=蒲桃因其木质的粉色而独一无二。|伐木工
+extratrees.block.planks.brazilwood.desc=巴西红木因其木质上紫色的阴影而出名。|伐木工
+extratrees.block.planks.logwood.desc=你可以根据一些树来找到红木，墨水树就是这其中之一。|伐木工
+extratrees.block.planks.iroko.desc=其富有吸引力的木材将会是过梁的上好材料。|伐木工
+extratrees.block.planks.locust.desc=和毁灭性的虫害毫无关联。|伐木工
+extratrees.block.planks.eucalyptus.desc=一种粉色木材，经常拿来制造木浆。|伐木工
+extratrees.block.planks.purpleheart.desc=此树正是得名于木质上紫色的阴影。|伐木工
+extratrees.block.planks.ash.desc=Gotta catch 'em all!|小智
+extratrees.block.planks.holly.desc=这是最白的木材，记得不要让有尖头的叶子掉进去。|伐木工
+extratrees.block.planks.olive.desc=那引人注目的柔和的绿色也正是其果实的色彩。|伐木工
+extratrees.block.planks.sweetgum.desc=又是一种质地均匀的木材，但它有一些用途。|伐木工
+extratrees.block.planks.rosewood.desc=这又是一种颜色颇深的木材，用于制作国际象棋棋子。|伐木工
+extratrees.block.planks.gingko.desc=和黄杨类似，银杏木颜色很淡。|伐木工
+extratrees.block.planks.pinkivory.desc=我能说些什么呢.它的粉色太浓厚了。|伐木工
+
+extratrees.block.planks.vanilla.oak.desc=
+extratrees.block.planks.vanilla.spruce.desc=
+extratrees.block.planks.vanilla.birch.desc=
+extratrees.block.planks.vanilla.jungle.desc=
+extratrees.block.planks.vanilla.darkoak.desc=
+extratrees.block.planks.vanilla.acacia.desc=
+
+extratrees.block.planks.forestry.larch.desc=
+extratrees.block.planks.forestry.teak.desc=
+extratrees.block.planks.forestry.acacia.desc=
+extratrees.block.planks.forestry.lime.desc=
+extratrees.block.planks.forestry.chestnut.desc=
+extratrees.block.planks.forestry.wenge.desc=
+extratrees.block.planks.forestry.baobab.desc=
+extratrees.block.planks.forestry.sequoia.desc=
+extratrees.block.planks.forestry.kapok.desc=
+extratrees.block.planks.forestry.ebony.desc=
+extratrees.block.planks.forestry.mahogany.desc=
+extratrees.block.planks.forestry.balsa.desc=
+extratrees.block.planks.forestry.willow.desc=
+extratrees.block.planks.forestry.walnut.desc=
+extratrees.block.planks.forestry.greenheart.desc=
+extratrees.block.planks.forestry.cherry.desc=
+extratrees.block.planks.forestry.mahoe.desc=
+extratrees.block.planks.forestry.poplar.desc=
+extratrees.block.planks.forestry.palm.desc=
+extratrees.block.planks.forestry.papaya.desc=
+extratrees.block.planks.forestry.pine.desc=
+extratrees.block.planks.forestry.plum.desc=
+extratrees.block.planks.forestry.maple.desc=
+extratrees.block.planks.forestry.citrus.desc=
+
+##################################################
+## Wood Blocks
+
+< %1$s is name of wood used for door >
+extratrees.block.door.name=%1$s木门
+
+< %1$s is name of wood used for door, %2$s is type of door >
+extratrees.block.door.name.adv=%1$s%2$s木门
+extratrees.block.door.type.standard=
+extratrees.block.door.type.solid=Solid
```

Comment:
Wut

Reply:
I don't know what are the meanings of these types of door and fence. So I give them up.

Reply:
Have a good idea?

Reply:
Well...
Others are ignored, so what about these?

Reply:
Well...
Others are ignored, so what about these?

Reply:
Well...
Others are ignored, so what about these?

Reply:
Well...
Others are ignored, so what about these?

Reply:
Well...
Others are ignored, so what about these?

Reply:
Well...
Others are ignored, so what about these?

Reply:
Well...
Others are ignored, so what about these?

Reply:
Well...
Others are ignored, so what about these?

Reply:
Well...
Others are ignored, so what about these?

Reply:
Well...
Others are ignored, so what about these?

Reply:
Well...
Others are ignored, so what about these?

Reply:
Forget it, ok?